examples/drivers/windows/otLwf/radio.c

```diff
@@ -814,6 +814,14 @@ void otPlatRadioSetDefaultTxPower(_In_ otInstance *otCtx, int8_t aPower)
     }
 }
 
+int8_t otPlatRadioGetReceiveSensitivity(_In_ otInstance *otCtx)
+{
+    NT_ASSERT(otCtx);
+    PMS_FILTER pFilter = otCtxToFilter(otCtx);
+    UNREFERENCED_PARAMETER(pFilter);
+    return -100;
```

Comment:
If you compare `platform/radio.h` and `link_raw.h` generally there is a one-to-one for all the APIs used by the core OpenThread code. `otPlatRadioGetRssi` isn't hooked up, because in the case of an offloaded energy scan (which Windows uses), the core code never calls that API. But in the case of this new API, it is being directly referenced.